stdcommands/undelete/undelete.go

```diff
@@ -32,6 +34,17 @@ var Command = &commands.YAGCommand{
 				}
 			}
 		}
+		
+		if targetUser != 0 {
+			if ok, err := bot.AdminOrPermMS(data.CS.ID, data.MS, discordgo.PermissionManageMessages); err != nil || !ok && data.MS.ID != targetUser {
```

Comment:
This line seems a bit complicated, it might read a little better splitting this into a few lines. No big deal though.